lib/jekyll-redirect-from/redirect_page.rb

```diff
@@ -21,6 +21,7 @@ def initialize(site, base, dir, name)
     def generate_redirect_content(item_url)
       self.output = self.content = <<-EOF
 <!DOCTYPE html>
+<html lang="en-US">
```

Comment:
@parkr I did, but since the text on the page is English (regardless of what the site language is), it felt like `en-US` was the best fit. Curious to hear any other thoughts about this though.